app/pages/editorg/template.njk

```diff
@@ -0,0 +1,54 @@
+{% extends 'includes/layout.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/button/macro.njk' import button %}
+{% from 'components/checkboxes/macro.njk' import checkboxes %}
+{% from '../../common/components/table.njk' import table %}
+
+{% block body %}
+  {% if agreementSigned %}
+    {% set checkedValue %}checked{% endset %}
+  {% endif %}
+
+  <div class="nhsuk-width-container" data-test-id="edit-organisation-page">
+    <div data-test-id="go-back-link">
+      {{ backLink({
+        "href": backLinkHref,
+        "text": "Back"
+      }) }}
+    </div>
+  
+    <h1 data-test-id="edit-org-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">Edit {{ organisationName }}</h1>
+    <h2 data-test-id="edit-org-page-description" class="nhsuk-u-font-size-22 nhsuk-u-margin-bottom-9">{{ description }}</h2>
+
+    <h3 data-test-id="organisation-ods-code-heading" class="nhsuk-heading-s nhsuk-u-margin-bottom-3">{{ odsCodeHeading }}</h3>
+    <div data-test-id="organisation-ods-code" class="nhsuk-u-font-size-19 nhsuk-u-margin-bottom-5">{{ odsCode }}</div>
```

Comment:
One for later but we seem to be doing this quite often. Perhaps a little component that has the above 2 lines

Reply:
can we create a tech debt ticket for this one please so we don't forget

Reply:
sure go ahead 😄